docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+
+Since the [RFC on localization](https://github.com/gatsbyjs/rfcs/pull/42) for Gatsby, there is now an ongoing effort to translate the content on Gatsbyjs.org into multiple languages. For members of the community around the world building Gatsby sites and learning about web development, having docs and learning materials in your own language provides a lot of value. There's also a great opportunity for folks like you to contribute to the translations of the Gatsby docs.
+
+If you're a native speaker of a language other than English and you're interested in helping translate gatsbyjs.org, here is some information for you to know:
+
+## The general process
+
+Each translation has its own repository in the [gatsbyjs](https://github.com/gatsbyjs/) organization on GitHub, with designated codeowners to review and approve changes. This doc will outline how to request a new language repository with its own team, what decisions each team can make, and suggestions for how you can work together.
+
+The Gatsby learning team is in charge of determining priorities for which docs should be translated. Refer to the [i18n page spreadsheet](https://docs.google.com/spreadsheets/d/1u2amGnqFLKxJuL5h9UrDblUueFgg0EBt7xbau4n8iTM/edit) to get the most up-to-date priority list.
+
+## Use English as the source
+
+The [gatsbyjs.org](https://gatsbyjs.org) website is written in English and should be considered the source material for all translations (as opposed to translating from another translation). When a repository is created, it will provide the space to copy over prioritized docs and [make pull requests](/contributing/how-to-open-a-pull-request/) against them in the relevant language.
+
+## Process for starting a new translation
+
+The first step for starting a new translation is to check what exists. So far, there are repositories for the following languages:
+
+- [Spanish](https://github.com/gatsbyjs/gatsby-es)
+- [Russian](https://github.com/gatsbyjs/gatsby-ru)
+
+You can find the list of all translation requests here: https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+%22New+Translation+Request%22+
+
+> Note: Once a new translation repository is created, feel free to add it here in a PR!
+
+### Finding codeowners
+
+For a new translation, open an issue with information about your intended language. If you already have co-contributors to act as fellow [code owners](https://help.github.com/en/articles/about-code-owners) and provide checks and balances for PR reviews and quality assurance, that would be very helpful! Otherwise, you can check out other [translation request issues](https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+%22New+Translation+Request%22) people have made and offer to join, or [get in touch with us](/contributing/how-to-contribute/#not-sure-how-to-start-contributing) at Gatsby for help in building your translation team.
+
+## Contributing to a translation
```

Comment:
We create a how-to contribute document to start with the translations here: https://github.com/gatsbyjs/gatsby-es/blob/master/CONTRIBUTING.MD.

The idea is that this document exposes how to start, create the fork, create a branch, open a PR, add labels and request for review, as well as the process to ask for the page they want to translate in the progress issue, but the point is that a document in the original language on `How to contribute` should be added because we consider that not only people that know English should contribute, we ask for people with really good Spanish, for example, to help curate content, like grammar or accents :muscle:

Reply:
Thanks @Darking360, I'm adding a link to it as an example!